source/content/guides/disaster-recovery/01-introduction.md

```diff
@@ -17,6 +17,6 @@ product: [--]
 integration: [--]
 ---
 
-This guide is focused on the immediate actions a Pantheon customer should take in the event of a catastrophic site failure. In all cases, the first step should be to file an emergency downtime on-call ticket, regular ticket, or open a support chat depending on support tier. Filing an emergency Pantheon On-Call ticket will immediately escalate the incident and ensure the fastest and most effective service.
+This guide is focused on the immediate actions a Pantheon customer should take in the event of a catastrophic site failure. In all cases, the first step should be to file an emergency ticket, a general support ticket, or start a live chat depending on your [account tier](/guides/support/#support-features-and-response-times). Filing an emergency ticket will immediately escalate the incident to our engineering team, and ensure the fastest and most effective service.
```

Comment:
We need to update this to be "support engineering" team, we cannot refer to "engineering" without the "support" qualifier. 

Customers, especially tenured ones, may think the ticket will go straight to PDE, when in reality it goes to Support.